src/applications/_mock-form-ae-design-patterns/sass/custom.scss

```diff
@@ -14,6 +14,27 @@
   flex: 1 1 auto;
 }
 
+[data-location="2/task-blue/veteran-information/edit-home-phone"] {
+.schemaform-first-field {
+  margin-top: 1.875rem !important;
+}
+}
+
+[data-testid="mini-summary-card"] {
+  @include media($medium-screen) {
+    width: 18.8rem;
+    height: 8.8rem;
+  }
+  &:nth-of-type(4) {
+    @include media($medium-screen) {
+      height: 11.4rem;
+    }
+  }
+}
```

Comment:
non-blocking minor nit: a class name might be better to use here. the data-testid attr is usually only used for testing in react testing library because it isn't something that is often changed, and is not typically tied to styling.

id say that same string of mini-summary-card could work just fine for a class name instead

Reply:
could we just call it 'summary-card' for succinctness?